docker-compose.prod.yml

```diff
@@ -1,95 +0,0 @@
-version: "2"
-
-services:
-  postgres:
-    image: postgres:9.4
-    volumes:
-      - "./build/data/postgres:/var/lib/postgresql/data"
-      - "./build/logs/postgres:/var/log"
```

Comment:
I suggest always providing the persistent volumes support, both in the development and production mode. Sometimes you generate data and local accounts and you don't want this to be gone after your battery dies or you forgot to update the production YAML file accidentally.